packages/calcite-components/src/components/combobox/combobox.e2e.ts

```diff
@@ -1632,7 +1632,7 @@ describe("calcite-combobox", () => {
     let element: E2EElement;
     let comboboxItem: E2EElement;
     let itemNestedLi: E2EElement;
-    let closeEvent: Promise<void>;
+    let closeEvent: Promise<unknown>;
```

Comment:
Stencil's waitForEvent has return type Promise<any> so this worked.
In Lumina, the return type is Promise<SerializedEvent>.

(at runtime, Stencil's waitForEvent also returned Promise<SerializedEvent>, but they didn't publicly expose the SerializedEvent type)